latch_cli/services/get_params.py

```diff
@@ -44,13 +44,29 @@ class Unsupported:
     bool: False,
 }
 
+# TODO(ayush): fix this to
+# (1) support records,
+# (2) support fully qualified workflow names,
```

Comment:
i'm pretty sure we already support fully qualified workflow names (`latch get-params wf.__init__.alphafold_wf`)